### PR TITLE
E2e rx misc unless

### PR DESCRIPTION
**Pull request by Droogans**
I'm tired of copying this function from various `util` directories between projects.

I also did a touch of refactoring in the rxMisc midway tests.

### CODE REVIEW

File: src/rxMisc/rxMisc.page.js [hunk 1]
```diff
@@ -45,5 +45,79 @@ exports.rxMisc = {
...
+    /**
+       If the `elem` is found, invoke `innerFn` on the resulting element's text.
+       If there is no element found (or displayed), return `null`.
+       If the element is found and displayed, but the text matches something in {@link rxMisc.nullValueMatches},
+       then return `null`.
+       This is useful when applications feature use of `ng-if` to control various messages to the user.
+       @function
+       @param {WebElement} elem - The web element that may or may not be present, displayed, and valid.
+       @param {Function} innerFn - Function to call on the element's text should it be present, displayed, and valid.
+       @param {*} [fallbackReturnValue=null] - Returned if the web element is not present, not displayed, or invalid.
```
Comment (parlarjb) [inline src/rxMisc/rxMisc.page.js]: can you put an `or` in front of `not displayed`, i.e.

`Returned if the web element is not present, or not displayed, or invalid.`

**Comment by Droogans** [comment]
Consider the PR builder passed....need to figure out these intermittent SAREs from rxNotify...

**Comment by Droogans** [comment]
@parlarjb can this get merged? The failing test is a fluke.

**Comment by parlarjb** [comment]
LGTM after a squash

**Comment by Droogans** [comment]
I kept the first commit separate since it did some refactor work.

**Pull request merged**